pvlib/modelchain.py

```diff
@@ -328,26 +328,26 @@ class ModelChain:
 
     dc_model: None, str, or function, default None
         If None, the model will be inferred from the contents of
-        system.module_parameters. Valid strings are 'sapm',
+        system.arrays[i].module_parameters. Valid strings are 'sapm',
         'desoto', 'cec', 'pvsyst', 'pvwatts'. The ModelChain instance will
         be passed as the first argument to a user-defined function.
 
     ac_model: None, str, or function, default None
         If None, the model will be inferred from the contents of
-        system.inverter_parameters and system.module_parameters. Valid
-        strings are 'sandia', 'adr', 'pvwatts'. The
+        system.inverter_parameters and system.arrays[i].module_parameters.
```

Comment:
I don't see that `infer_ac_model` uses any information from `system.arrays`. The check on `num_arrays` is meant to provide a more meaningful error message, if a user chooses the `adr` inverter model that doesn't (presently) support multiple inputs.